Epsilon.Host.Frontend/src/components/Competence/CompetenceProfile.vue

```diff
@@ -4,15 +4,17 @@
 			<tr>
 				<td />
 				<th
-					v-for="col of store.state.domain.columnsSet?.types"
+					v-for="col of store.state.domain.columnsSet?.types.sort((a: LearningDomainType, b: LearningDomainType) => (a?.order || 0) - (b?.order || 0))"
 					:key="col.id"
 					class="competence-profile-header competence-profile-header-col">
 					{{ col.name }}
 				</th>
 			</tr>
 		</thead>
 		<tbody>
-			<tr v-for="row of store.state.domain.rowsSet?.types" :key="row.id">
+			<tr
+				v-for="row of store.state.domain.rowsSet?.types.sort((a: LearningDomainType, b: LearningDomainType) => (a?.order || 0) - (b?.order || 0))"
```

Comment:
```suggestion
				v-for="row of store.state.domain.rowsSet?.types.sort((a) => a.order ?? 0)"
```

Reply:
Als je dit zo schrijft dan krijg je errors met front-end linting, aangezien de variable undefined is. Ook worden de rows niet gesorteerd.